### DISCUSSION TITLE
Opensource

### DISCUSSION THREAD

**Discussion by lahiruroot** [lahiruroot/WUSL#6] [Q&A] [answered]
Give a opensource free software

**Comment by PrasadWc** [comment, marked as answer]
Linux

**Comment by devinda6655** [comment]
Linux

**Comment by KavinduYohan** [comment]
Linux

**Comment by ThilinaPrabhasara** [comment]
ubuntu

**Comment by Kavithaangi** [comment]
Linux

**Comment by SuloSew** [comment]
LibreOffice: A free alternative to Microsoft Office for creating documents, spreadsheets, and presentations.


GIMP: A powerful open-source image editing software similar to Photoshop.


Audacity: Free and open-source software for audio recording and editing.


Blender: A comprehensive tool for 3D modeling, animation, and rendering.

**Comment by ReshaniDananjana** [comment]
Linux

**Comment by Chathushka04** [comment]
VS Code (Visual Studio Code)

**Comment by sheshanCharuka** [comment]
Linux
